### PR TITLE
#431 improve configsource api

### PR DESCRIPTION
**Pull request by Emily-Jiang**
Signed-off-by: Emily Jiang <emijiang6@googlemail.com>

### CODE REVIEW

File: tck/src/main/java/org/eclipse/microprofile/config/tck/CDIPlainInjectionTest.java [hunk 1]
```diff
@@ -104,36 +106,34 @@ public void canInjectSimpleValuesWhenDefined() {
...
+        assertThat(bean.longProperty, is(equalTo(ConfigProvider.getConfig().getValue("my.long.property", Long.class))));
+        assertThat(bean.floatProperty,
+                is(floatCloseTo(ConfigProvider.getConfig().getValue("my.float.property", Float.class), 0.1f)));
+        assertThat(bean.doubleProperty,
+                is(closeTo(ConfigProvider.getConfig().getValue("my.double.property", Double.class), 0.1)));
+        assertThat(bean.charProperty,
+                is(equalTo(ConfigProvider.getConfig().getValue("my.char.property", Character.class))));
+
+        assertThat(bean.doublePropertyWithDefaultValue, is(closeTo(ConfigProvider.getConfig()
+                .getOptionalValue("my.not.configured.double.property", Double.class).orElse(3.1415), 0.1)));
```
Comment (dmlloyd) [inline tck/src/main/java/org/eclipse/microprofile/config/tck/CDIPlainInjectionTest.java]: What is the actual change here?  Was this just reformatted, and if so could it be reverted?

Comment (Emily-Jiang) [reply tck/src/main/java/org/eclipse/microprofile/config/tck/CDIPlainInjectionTest.java]: For some reason, my reformatting tool kicked in. Cleaned up now.

Comment (radcortez) [reply tck/src/main/java/org/eclipse/microprofile/config/tck/CDIPlainInjectionTest.java]: Maybe we could adopt https://github.com/revelc/formatter-maven-plugin? Actually I'll submit an issue with it.

**Comment by Emily-Jiang** [comment]
@dmlloyd I've addessed your comments.

**Pull request merged**